lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -271,7 +271,10 @@
   user:
     required: false
     default: root
-    description: username or UID
+    description:
+    - username or UID.
+    - "If set to C(none) the docker daemon default value (or the one already
```

Comment:
The value must be `null` (that's YAML's `None`), not `none`. It *might* be that Ansible does some magic to also convert `"none"` to `None`, but I guess it's safer to use the correct YAML keyword.

Reply:
Thanks, I also wrote the same value in the changelog fragment.